content/billing/billing.md

```diff
@@ -43,8 +45,12 @@ For pay-as-you-go team plans, each team user is billed at the rate on the [prici
 
 For pay-as-you-go team and user plans, each build minute is billed at the rate on our [pricing page](https://codemagic.io/pricing/) based on the build [machine type](../specs/machine-type) used for the build. Builds that time out or fail because of a Codemagic service error will not count towards billing usage. Builds that fail for any other reason will count towards billing usage.
 
-## Overdue invoices
+## Overdue and failed invoice payments
+
+Invoices that have not been paid on the due date are shown as "Overdue" or "Failed" on the [Billing](https://codemagic.io/billing/) page. This usually happens when there are problems with charging the card, of which team or account owners are notified by email. Codemagic will then attempt to retry the payment in 3, 5 or 7 days.
 
-Invoices that have not been paid on the due date are shown as "Overdue" on the [Billing](https://codemagic.io/billing/) page. This usually happens when there are problems with charging the card, of which team or account owners are notified by email. Codemagic will then attempt to retry the payment in 3, 5 or 7 days. If all retries for a payment fail, the subscription is cancelled.
+As soon as there is a failed invoice payment, your subscription will be put on hold. In order to continue building all invoices should be paid.
 
 Team or account owners can manually retry the payment by clicking on the overdue invoice in **Billing history** and selecting **Pay invoice** on the popup with invoice details.
+
+Note that, overdue invoices should be paid before subscription could be disabled. Please refer to [Disable billing](/billing/billing/#disabling-billing) section for further info.
```

Comment:
Remove comma after "Note that"
.. before billing can be disabled.
Please refer to the Disabling billing section...